FourthTask/figures/main/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/tests"
```

Comment:
It's better to keep snake_case in packages in general. e.g. it's OK to use upper case for some abbreviations.

Reply:
I'll consider it in next hometask.